helm/superset/templates/deployment-worker.yaml

```diff
@@ -130,6 +135,10 @@ spec:
         - name: superset-config
           secret:
             secretName: {{ tpl .Values.configFromSecret . }}
-      {{- with .Values.extraVolumes }}
-      {{- tpl (toYaml .) $ | nindent 8 -}}
-      {{- end }}
+        - name: superset-extra-config
```

Comment:
Looks like this is missing a leading `{{- if .Values.extraConfigs }}`

Reply:
good catch, i just missed it :+1: